java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java

```diff
@@ -863,4 +869,60 @@ public static void setMinValidationLevel(IsolateThread thread, ObjectHandle netw
         });
     }
 
+    @CEntryPoint(name = "attachNewLineOnLine")
+    public static void attachNewLineOnLine(IsolateThread thread, ObjectHandle networkHandle,
+                                           CCharPointer voltageLevelId,
+                                           CCharPointer bbsIdBusId,
+                                           CCharPointer lineId,
+                                           double percent,
+                                           DataframePointer cDataframeLine,
+                                           ExceptionHandlerPointer exceptionHandlerPtr) {
+        doCatch(exceptionHandlerPtr, () -> {
+            String voltageLevelIdStr = CTypeUtil.toString(voltageLevelId);
+            String bbsIdBusIdStr = CTypeUtil.toString(bbsIdBusId);
+            String lineIdStr = CTypeUtil.toString(lineId);
+            Network network = ObjectHandles.getGlobal().get(networkHandle);
+            Line line = network.getLine(lineIdStr);
+            UpdatingDataframe df = createDataframe(cDataframeLine);
+            LineAdder adder = network.newLine();
+            applyIfPresent(df.getStrings("id"), 0, adder::setId);
+            applyIfPresent(df.getDoubles("b1"), 0, adder::setB1);
+            applyIfPresent(df.getDoubles("b2"), 0, adder::setB2);
+            applyIfPresent(df.getDoubles("g1"), 0, adder::setG1);
+            applyIfPresent(df.getDoubles("g2"), 0, adder::setG2);
+            applyIfPresent(df.getDoubles("r"), 0, adder::setR);
+            applyIfPresent(df.getDoubles("x"), 0, adder::setX);
+            CreateLineOnLine createLineOnLine = new CreateLineOnLineBuilder()
+                    .withBusbarSectionOrBusId(bbsIdBusIdStr)
+                    .withLine(line)
+                    .withPercent(percent)
+                    .withLineAdder(adder)
+                    .build();
+            createLineOnLine.apply(network);
```

Comment:
constructor of CreateLineOnLine has a more lot attributes 
```java
public CreateLineOnLine(double percent, String voltageLevelId, String bbsOrBusId, String fictitiousVlId, String fictitiousVlName, boolean createFictSubstation, String fictitiousSubstationId, String fictitiousSubstationName, String line1Id, String line1Name, String line2Id, String line2Name, Line line, LineAdder lineAdder)
```
Do I add them ?

Reply:
Note sure if that's relevant. It will probably makes for a heavier API on pypowsybl side. @annetill Do you have an opinion on some of these parameters ?

Reply:
I think its related to the fictitious substation creation options and the optionnal renaming of the two new lines created by cutting the existing one.